crossing-training.lic

```diff
@@ -89,6 +96,7 @@ class CrossingTraining
     if trash_nouns.any? { |noun| /\b#{noun}/i =~ GameObj.left_hand.noun } && !@equipment_manager.is_listed_item?(left_hand)
       dispose_trash(left_hand)
     end
```

Comment:
Plucked it out of a hat. A lower number seems good to keep the trash down though. Change it to 13?